api/sheets/get_variable.md

```diff
@@ -1,111 +1,119 @@
 ---
 layout: default
-title: Statistical Variable Value
+title: Returning Statistical Variables
 nav_order: 3
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the value of a statistical variable at a given place and time
+# Returning Statistical Variables
 
-Given a list of [DCIDs](/glossary.html) representing [Place](https://datacommons.org/browser/Place) objects, a [StatisticalVariable](https://datacommons.org/browser/StatisticalVariable), and optionally a date, get the measurements of the specified variable in the specified places with a date (if specified).
+The`=DCGET(dcids, variable, date)` formula returns the measurements of a specified [statistical variable](https://docs.datacommons.org/glossary.html#variable) at a given place and optional time based on a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html). A complete list of variables can be found in the [graph browser](https://datacommons.org/browser/StatisticalVariable).
 
-## General information about this formula
+> **Note**:
+> Be sure to follow the instructions for Installing and Enabling the Sheets Add-On before using this formula.
 
-**Formula**: `=DCGET(dcids, variable, date)`
+## Formula
 
-**Required arguments**:
+=DCGET(*dcids*, *variable*, *date*)
 
-*   [`dcids`](/glossary.html): A list of `Place` nodes, identified by their DCIDs.
+## Required Arguments
 
-*   `variable` - The [StatisticalVariable](/glossary.html) whose measurement is sought.
+● [dcids](https://docs.datacommons.org/glossary.html): A list of `Place` nodes, identified by their DCIDs.
```

Comment:
please use `*` or `-` to get markdown bullets

Reply:
please also use relative URL paths

Reply:
OK